examples/certificate_signing_example.yml

```diff
@@ -0,0 +1,46 @@
+---
+- name: Example - how to generate private keys and sign certificates with openSSL
```

Comment:
`Example - how to generate private keys and sign certificates with openSSL and upload to HyperCore server`

Reply:
Changed.